src/Module/Special/HTTPException.php

```diff
@@ -7,13 +7,16 @@
 
 namespace Friendica\Module\Special;
 
-use Friendica\App;
+use Friendica\App\Arguments;
+use Friendica\App\Request;
 use Friendica\Core\L10n;
 use Friendica\Core\Renderer;
 use Friendica\Core\Session\Model\UserSession;
 use Friendica\Core\System;
 use Friendica\Module\Response;
+use Friendica\Network\HTTPException as NetworkHTTPException;
```

Comment:
Why the need for an alias?

Reply:
Because we are already in a class also called `(Special\) HTTPException`.

Reply:
Aaaaah, right, good catch.